web/src/popover_menus.js

```diff
@@ -386,7 +386,7 @@ export function initialize() {
     // that could possibly obstruct user from using this popover.
     delegate("body", {
         ...default_popover_props,
-        target: ".compose_mobile_button",
+        target: ".mobile_button_container",
```

Comment:
Probably worth a comment noting that we're picking this rather than `compose_mobile_button` because we can't have both a popover and a tooltip registered to the same element.

Reply:
Comment added!